backend/src/common/helpers/crypt-compare/crypt-compare.helper.ts

```diff
@@ -0,0 +1,10 @@
+import { compare } from 'bcrypt';
+
+const cryptCompare = async (
+    data: string,
+    encrypted: string,
+): Promise<boolean> => {
+    return await compare(data, encrypted);
```

Comment:
You don't need to _return await_ 
I have good article about it http://hassansin.github.io/Why-return-await-Is-a-Bad-Idea